manifests/artemis.pp

```diff
@@ -0,0 +1,58 @@
+# Configuration for Artemis
+#
+# @api private
+class candlepin::artemis {
+  assert_private()
+
+  file { "${candlepin::catalina_home}/conf":
+    ensure => directory,
+    mode   => '0755',
+    owner  => 'tomcat',
+    group  => 'tomcat',
+  } ~>
+  file { "${candlepin::catalina_home}/conf/conf.d":
+    ensure => directory,
+    mode   => '0755',
+    owner  => 'tomcat',
+    group  => 'tomcat',
+  } ~>
+  file { $candlepin::broker_config_file:
+    ensure  => file,
+    content => template('candlepin/broker.xml.erb'),
+    mode    => '0640',
+    owner   => 'tomcat',
+    group   => 'tomcat',
+  } ~>
+  file { "${candlepin::catalina_home}/conf/login.config":
+    ensure  => file,
+    content => template('candlepin/tomcat/login.config'),
+    mode    => '0640',
+    owner   => 'tomcat',
+    group   => 'tomcat',
+  } ~>
+  file { "${candlepin::catalina_home}/conf/cert-users.properties":
+    ensure  => file,
+    content => template('candlepin/tomcat/cert-users.properties'),
+    mode    => '0640',
+    owner   => 'tomcat',
+    group   => 'tomcat',
+  } ~>
+  file { "${candlepin::catalina_home}/conf/cert-roles.properties":
+    ensure  => file,
+    content => template('candlepin/tomcat/cert-roles.properties'),
```

Comment:
If it's a single line (as it is now), you don't need to use a template.
```suggestion
    content => "candlepinEventsConsumer=katelloUser\n",
```

Reply:
That's true, but what is the downside of using a file and having a consistent way of settings these.

Reply:
The downside of using `file()` is that (as a human) you need to open another file to see the content while inlined you have a complete view. At runtime there's probably a very small overhead of opening another file as well, but IMHO that's irrelevant. If it's a few lines, you can also consider [heredoc](https://github.com/puppetlabs/puppet-specifications/blob/master/language/heredoc.md).